Expensify/App
# Added tooltip for the communications link

### Details


### Fixed Issues

$ https://github.com/Expensify/App/issues/5542

### Tests

1. Tested with long emails
2. Tested for mobile layouts to ensure nothing breaks

### QA Steps

1. Navigate to a conversation with a user that has a long email
2. Open the user detail page
3. Hover the cursor over the email and you should see a tooltip

### Tested On


### Screenshots


#### Web

<img width="578" alt="Screenshot 2021-11-01 at 2 57 31 PM" src="https://user-images.githubusercontent.com/57435789/139652189-5d7bef28-528d-4d6b-8804-d4d94913793b.png">


#### Mobile Web

NA

#### Desktop

<img width="519" alt="Screenshot 2021-11-01 at 3 06 36 PM" src="https://user-images.githubusercontent.com/57435789/139652210-7463bc88-4d53-4196-aa6d-edd6b9abc111.png">


#### iOS

NA

#### Android

NA
